api/server.js

```diff
@@ -0,0 +1,45 @@
+
+//IMPORTS
+const express = require( 'express' );
+const server = express();
+
+//IMPORT THE GUARDIANS OF THE GALAXY
+const guardians = require( '../data/guardians/guardians' );
+
+//MIDDLEWARE
+server.use( express.json());
+
+//HOME PAGE
+server.get( '/' , async ( req , res ) => {
+    res.status( 200 ).json({ message: 'GUARDIANS OF THE GALAXY!' })
```

Comment:
no need to make this one an async method as you are not utilizing await anywhere